scripts/deploy.mjs

```diff
@@ -24,9 +24,9 @@ export async function setupContracts() {
   //   throw new Error("shared nft logic address is required");
   // }
 
-  if (!creatorProxyAddress) {
-    throw new Error("creator proxy address is required");
-  }
+  // if (!creatorProxyAddress) {
+  //   throw new Error("creator proxy address is required");
+  // }
```

Comment:
do we still need this?

Reply:
nah I will remove 🚮

Reply:
I'm gonna make this a PR with the base repo (owned by Zora) 📚 
- this variable isn't used and we can help cleanup the zora codebase ✨

Reply:
super tiny PR to say hi to ZoraENG 👋

Reply:
removed from this PR as well. 🚮 ✅